src/Backups/IBackupCoordination.h

```diff
@@ -85,6 +90,20 @@ class IBackupCoordination
 
         /// Position in the archive.
         UInt64 pos_in_archive = static_cast<UInt64>(-1);
+
+        [[ maybe_unused ]] String describe()
```

Comment:
This format cannot be parsed back, a comment is needed to explain that the method is only suitable for debugging.